resolwe_bio/processes/junctions/regtools.yml

```diff
@@ -0,0 +1,119 @@
+# =========================================
+# Regtools - Junctions Extract and Annotate
+# =========================================
+---
+
+- slug: junctions-annotate
+  name: Annotate novel splice junctions (regtools)
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq:regtools
```

Comment:
Remember to replace this with the correct numerical tag when the resolwebio/rnaseq Docker image includes the regtools tool